rasa/core/training/interactive.py

```diff
@@ -707,7 +707,7 @@ def _request_export_info() -> Tuple[Text, Text, Text]:
             "will append the stories)",
             default=PATHS["stories"],
             validate=io_utils.file_type_validator(
-                [".md"],
+                [".md", ".yml"],
```

Comment:
We can use `YAML_FILE_EXTENSIONS + MARKDOWN_FILE_EXTENSIONS` from `rasa.shared.data` here.